lib/src/widgets/overlay.dart

```diff
@@ -358,7 +359,9 @@ class _DescribedFeatureOverlayState extends State<DescribedFeatureOverlay>
     _close();
   }
 
-  Future<void> _dismiss() async {
+  /// If [force] is true, the [onDismiss] function will be awaited
```

Comment:
If we want to have `dismissAll`, it should be done properly. Convince me that this is needed.  
I think that this could lead to some unexpected behavior, e.g. if you have a future that does not complete straight away, but you want to forcefully dismiss all overlays